frontend/src/modules/DistributionPlot/view.tsx

```diff
@@ -17,9 +19,20 @@ import { makeHistogramTrace } from "./utils/histogram";
 import { makeHoverText, makeHoverTextWithColor, makeTitleFromChannelContent } from "./utils/stringUtils";
 import { calcTextSize } from "./utils/textSize";
 
+const MaxNumberPlotsExceededMessage: React.FC = () => {
+    return (
+        <ContentWarning>
+            <Warning fontSize="large" className="mb-2" />
+            Too many plots to display. Due to performance limitations, the number of plots is limited to 12.
```

Comment:
Make: `const MAX_NUMBER_OF_PLOTS = 12;`
Make the number a prop for the `MaxNumberPlotsExceededMessage`-component and use the `MAX_NUMBER_OF_PLOTS` for the checks below.